pages/blog/miniseries-part_1.md

```diff
@@ -0,0 +1,227 @@
+---
+title: "Mini series - Speed up your development process (part 1)"
+date: 2021-04-28T06:00:00+01:00
+type: Engineering
+tags:
+  - tools
+  - code generation
+cover: /img/posts/jonaslagoni-miniseries-part1/blog-miniseries-cover.webp
+authors:
+  - name: Jonas Lagoni
+    photo: /img/avatars/jonaslagoni.webp
+    link: https://github.com/jonaslagoni
+    byline: AsyncAPI Maintainer
+featured: true
+---
+
+How can you utilize code generation to speed up the development process and only focus on what is important, the business logic? In this mini-series we will explore the ways AsyncAPI and code generation can work hand in hand beyond generating documentation. 
+
+Structure of the miniseries:
+* **Part 1: Designing the API's with AsyncAPI**
```

Comment:
doesn't match the title, is it intentional?

Reply:
Kinda, not sure how they can be synced? As `Designing the API's with AsyncAPI (part 1)` as the title seems weird to me like it is part 1 of how to design your API's with AsyncAPI when it is part 1. Not sure, what do you think? 🤔

Reply:
I think `Designing the API's with AsyncAPI` is better than `Speed up your development process`

Reply:
True. I'll change it.